ssz/utils.py

```diff
@@ -12,12 +13,26 @@ def is_sedes(obj):
     return hasattr(obj, 'serialize') and hasattr(obj, 'deserialize')
 
 
-def infer_sedes(obj):
+def get_integer_serializer(serializer_type):
+    '''
+    Given serializer_type as 'int<N>' or 'uint<N>', this function
+    returns the corresponding serializer object.
+    '''
+    integer_serializer = Integer(serializer_type)
+    return integer_serializer
+
+
+def infer_sedes(obj, serializer_type=None):
     """
     Try to find a sedes objects suitable for a given Python object.
+    :param serializer_type: str
+    The serializer_type parameter is optional and can be only
+    of the format 'int<N>' or 'uint<N>'(as of now)
     """
     if isinstance(obj, bool):
         return boolean
+    elif isinstance(obj, int):
+        return get_integer_serializer(serializer_type)
```

Comment:
Super *slightly* optimization for eth2.0 use case: maybe swap the order of `isinstance(obj, int)` and `isinstance(obj, bool)`.
```python
    if isinstance(obj, int):
        return get_integer_serializer(serializer_type)
    elif isinstance(obj, bool):
        return boolean
```

Reply:
@hwwhww but if the code was refactored as above, the following bug would occur where `bool` type would be regarded as `int`. Because as per `python3.6`
```
isinstance(True, int) = True
```